web/modules/custom/ai_screening_project_track/src/Helper/ProjectTrackToolHelper.php

```diff
@@ -356,6 +358,48 @@ public function getToolStatus(ProjectTrackToolInterface $tool): array {
     return $status;
   }
 
+  /**
+   * Get blockers for a tool.
+   */
+  public function getToolBlockers(ProjectTrackToolInterface $tool): array {
+    if ($tool->getToolEntityType() !== 'webform_submission') {
+      return [];
+    }
+
+    $toolId = $tool->getToolId();
+    $toolData = $tool->getToolData();
+
+    if (empty($toolData) || empty($toolId)) {
+      return [];
+    }
+
+    $webform = $toolData['webform_submission:' . $toolId]['webform'];
+    $submission = $toolData['webform_submission:' . $toolId]['submission'];
+
+    $webformFromConfig = Webform::create([
+      'elements' => Yaml::encode($webform),
+    ]);
+
+    $elements = $webformFromConfig->getElementsDecodedAndFlattened();
+
+    // We are only looking for ai_screening_yes_no_stop elements.
+    foreach ($elements as $key => $element) {
+      if ('ai_screening_yes_no_stop' !== $element['#type']) {
+        unset($elements[$key]);
+      }
+    }
```

Comment:
Consider using `array_filter` for this:

```suggestion
    $elements = array_filter($elements, static fn (array $element) => 'ai_screening_yes_no_stop' === ($element['#type'] ?? NULL));
```

Not all elements have a `#type` key (we've struggled with this is OS2Forms); therefore we must use `$element['#type'] ?? NULL` to get the type.